test/selenium-lib.js

```diff
@@ -21,7 +21,8 @@ function buildDriver() {
     return sharedDriver;
   }
   // Firefox options.
-  // http://selenium.googlecode.com/git/docs/api/javascript/module_selenium-webdriver_firefox.html
+  // http://selenium.googlecode.com/git/docs/api/javascript/
+  //  module_selenium-webdriver_firefox.html
```

Comment:
http://eslint.org/docs/rules/max-len.html#options this rule has options for ignoring URLs.